sport/app/football/controllers/FixturesController.scala

```diff
@@ -52,53 +60,34 @@ class FixturesController(
   def tagFixturesFor(year: String, month: String, day: String, tag: String): Action[AnyContent] =
     renderTagFixtures(createDate(year, month, day), tag)
 
-  private def renderTagFixtures(date: LocalDate, tag: String): Action[AnyContent] = {
+  private def getTagFixtures(date: LocalDate, tag: String): Option[(FootballPage, Fixtures)] = {
     lookupCompetition(tag)
-      .map { comp =>
-        renderCompetitionFixtures(tag, comp, date)
-      }
+      .map(comp =>
+        (
+          new FootballPage(s"football/$tag/fixtures", "football", s"${comp.fullName} fixtures"),
+          CompetitionFixturesList(date, competitionsService.competitions, comp.id, tag),
+        ),
+      )
       .orElse {
-        lookupTeam(tag).map(renderTeamFixtures(tag, _, date))
-      }
-      .getOrElse {
-        Action(NotFound)
+        lookupTeam(tag).map(team =>
+          (
+            new FootballPage(s"football/$tag/fixtures", "football", s"${team.name} fixtures"),
+            TeamFixturesList(date, competitionsService.competitions, team.id, tag),
+          ),
+        )
       }
   }
 
-  private def renderCompetitionFixtures(
-      competitionName: String,
-      competition: Competition,
-      date: LocalDate,
-  ): Action[AnyContent] =
-    Action { implicit request =>
-      val fixtures = CompetitionFixturesList(date, competitionsService.competitions, competition.id)
-      val page =
-        new FootballPage(s"football/$competitionName/fixtures", "football", s"${competition.fullName} fixtures")
-      renderMatchList(page, fixtures, filters)
-    }
-
-  private def renderTeamFixtures(teamName: String, team: FootballTeam, date: LocalDate): Action[AnyContent] =
-    Action { implicit request =>
-      val fixtures = TeamFixturesList(date, competitionsService.competitions, team.id)
-      val page = new FootballPage(s"football/$teamName/fixtures", "football", s"${team.name} fixtures")
-      renderMatchList(page, fixtures, filters)
-    }
-
-  def teamFixturesComponentJson(teamId: String): Action[AnyContent] = teamFixturesComponent(teamId)
```

Comment:
I needed to update this method when refactoring `TeamFixturesList` but I noticed that these methods aren't referenced anywhere, not even in a routes file, so instead of updating them I've removed them.